src/main/java/com/powsybl/ws/commons/computation/service/AbstractWorkerService.java

```diff
@@ -116,16 +116,21 @@ protected boolean resultCanBeSaved(R result) {
         return result != null;
     }
 
+    protected abstract void addLogsWhenFailed(C runContext, AtomicReference<ReportNode> rootReportNode);
```

Comment:
Remove abstract and give default implementation. 
So that we don't need to implement it in every computation service.
Something like : 
protected void addLogsWhenFailed(C runContext, AtomicReference<ReportNode> rootReportNode) {throw new UnsupportedOperationException("Not implemented");} ?